compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -944,6 +945,85 @@ LogicalResult setScatterLoweringConfig(IREE::GPU::TargetAttr target,
       {flatWorkgroupSize, 1, 1}, flatWorkgroupSize, DictionaryAttr());
 }
 
+//====---------------------------------------------------------------------===//
+// Sort Pipeline Configuration
+//====---------------------------------------------------------------------===//
+
+LogicalResult setSortConfig(IREE::GPU::TargetAttr target,
+                            mlir::FunctionOpInterface entryPoint,
+                            Operation *op) {
+
+  assert(isa<IREE::LinalgExt::SortOp>(op) && "expected linalg_ext.sort op");
+  MLIRContext *context = op->getContext();
+  Builder b(context);
+
+  auto subgroupSize = target.getPreferredSubgroupSize();
+  auto interfaceOp = cast<PartitionableLoopsInterface>(*op);
+  SmallVector<unsigned> partitionedLoops =
+      interfaceOp.getPartitionableLoops(kNumMaxParallelDims);
+
+  auto createLoweringConfig = [&](SmallVector<int64_t> &workgroupSizes,
+                                  SmallVector<int64_t> &threadSizes) {
+    SmallVector<NamedAttribute, 2> attrs = {
+        NamedAttribute("workgroup", b.getI64ArrayAttr(workgroupSizes)),
+        NamedAttribute("thread", b.getI64ArrayAttr(threadSizes))};
+    auto configDict = b.getDictionaryAttr(attrs);
+    return IREE::GPU::LoweringConfigAttr::get(context, configDict);
+  };
+
+  if (partitionedLoops.empty()) {
+    SmallVector<int64_t> defaultTileSizes = {0};
+    IREE::GPU::LoweringConfigAttr loweringConfig =
+        createLoweringConfig(defaultTileSizes, defaultTileSizes);
```

Comment:
Here, you don't need a vector either. `ArrayRef` has a constructor that takes a single element.
```suggestion
    IREE::GPU::LoweringConfigAttr loweringConfig =
        createLoweringConfig(int64_t{0}, int64_t{0});
```